clouddriver-appengine/src/test/groovy/com/netflix/spinnaker/clouddriver/appengine/deploy/ops/DeployAppengineAtomicOperationSpec.groovy

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2017 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.appengine.deploy.ops
+
+import com.netflix.spinnaker.clouddriver.appengine.deploy.ops.DeployAppengineAtomicOperation
+import spock.lang.Specification
+
+class DeployAppengineAtomicOperationSpec extends Specification {
+  void "updates runtime field in app.yaml to be 'custom'"() {
+    setup:
+      def configs = [
+        "runtime: foobar",
+        '''
+        env: flex
+        runtime: python27
+        manual_scaling:
+          instances: 1
+        resources:
+          cpu: 1
+          memory_gb: 0.5
+          disk_size_gb: 10
+        '''
+
+      ]
+      List<String> results
```

Comment:
Would just have `List<String> results = DeployAppengineAtomicOperation.updateConfigRuntime(configs)` unless Spock for some reason is unhappy with that.